### PR TITLE
Freeze time in TestSubtaskState.test_to_dictionary

### LINKED ISSUE #2192: TestSubtaskState.test_to_dictionary is flaky
```
@staticmethod
    def test_to_dictionary():
        ss = SubtaskState()
        ss.subtask_definition = "My long task definition"
        ss.subtask_id = "ABCDEF"
        ss.subtask_progress = 0.92
        ss.time_started = get_timestamp_utc()
        ss.deadline = timeout_to_deadline(ss.time_started + 5)
        ss.extra_data = {"param1": 1323, "param2": "myparam"}
        ss.subtask_rem_time = deadline_to_timeout(ss.deadline) - ss.time_started
        ss.subtask_status = SubtaskStatus.starting
        ss.value = 138
        ss.stdout = "path/to/file"
        ss.stderr = "path/to/file2"
        ss.results = ["path/to/file3", "path/to/file4"]
        ss.computation_time = 130
        ss.computer.node_name = "NODE1"
        ss.computer.node_id = "abc131"
        ss.computer.performance = 180
        ss.computer.ip_address = "10.10.10.1"
        ss.computer.port = 1311
    
        ss_dict = ss.to_dictionary()
        assert ss_dict['description'] == "My long task definition"
        assert ss_dict['subtask_id'] == "ABCDEF"
        assert ss_dict['progress'] == 0.92
>       assert ss_dict['time_started'] <= time.time()
E       AssertionError: assert 1519291818.191131 <= 1519291818.1911309
E        +  where 1519291818.1911309 = <built-in function time>()
E        +    where <built-in function time> = time.time

```
Because it depends on system's `time`, use freeze_gun instead.

### PR DESCRIPTION
**Pull request by Wiezzel**
Without freeze this test case was flaky.

Closes #2192

### CODE REVIEW

File: tests/golem/task/test_taskstate.py [hunk 1]
```diff
@@ -43,12 +46,12 @@ def test_to_dictionary():
         assert ss_dict['description'] == "My long task definition"
         assert ss_dict['subtask_id'] == "ABCDEF"
         assert ss_dict['progress'] == 0.92
-        assert ss_dict['time_started'] <= time.time()
+        assert ss_dict['time_started'] == time.time()
```
Comment (Krigpl) [inline tests/golem/task/test_taskstate.py]: I wonder, in line 29 we use `get_timestamp_utc` and here it's `time.time` and those two aren't guaranteed to be the same. Maybe change this one to `get_timestamp_utc` as well.

**Review by Krigpl — APPROVED** [review]

File: tests/golem/task/test_taskstate.py (hunk 1 above)
Comment (Wiezzel) [reply tests/golem/task/test_taskstate.py]: `get_timestamp_utc` uses `datetime.now` which uses `time.time`. So how exactly is any difference possible here?

Comment (Krigpl) [reply tests/golem/task/test_taskstate.py]: `datetime.now` doesn't use `time.time`.

Comment (Wiezzel) [reply tests/golem/task/test_taskstate.py]: Really?
```python
@classmethod
def now(cls, tz=None):
    "Construct a datetime from time.time() and optional time zone info."
    t = _time.time()
    return cls.fromtimestamp(t, tz)
```

Comment (Krigpl) [reply tests/golem/task/test_taskstate.py]: I'm not gonna dig into whether `_time` is the same as `time` but the point is, `time.time` return seconds from epoch while `get_timestamp_utc` returns seconds from unix epoch, where epoch is system defined and unix epoch is not.

Comment (Wiezzel) [reply tests/golem/task/test_taskstate.py]: Yes, `_time` is the same as `time`.
```python
import time as _time
```

Comment (Wiezzel) [reply tests/golem/task/test_taskstate.py]: Ok, changed to `get_timestamp_utc`.

**Review by tworec — APPROVED** [review]
LGTM

**Pull request merged**